src/com/opc/opceventhandler.cpp

```diff
@@ -41,18 +66,18 @@ void COpcEventHandler::run(){
 
   if(result == S_OK){
     while(isAlive()){
-      ICmd* nextCommand = getNextCommand();
-      if(nextCommand != nullptr) {
-        nextCommand->runCommand();
-        delete nextCommand;
+      if (!isAlive()) {
+        break;
       }
-
-      MSG msg;
```

Comment:
@azoitl Sir, what exactly this MSG for? Is it related to OPC multithread?

Reply:
Unfortunately I don't know. I was not involved in the original development of our OPC DA infrastructure and never used it myself.